lib/task_helpers.rb

```diff
@@ -1,213 +1,108 @@
 require 'elasticsearch'
+require_relative '../models/comment'
+require_relative '../models/comment_thread'
 
 module TaskHelpers
   module ElasticsearchHelper
     LOG = Logger.new(STDERR)
+    INDEX_MODELS = [Comment, CommentThread]
+    INDEX_NAMES = [Comment.index_name, CommentThread.index_name]
```

Comment:
Does ruby have tuple like data structure?

Reply:
Done. **Useful [link](https://www.rubypigeon.com/posts/a-review-of-immutability-in-ruby/)**